docs/examples.rst

```diff
@@ -0,0 +1,183 @@
+Examples
+########
+
+Flow Production Tracking Jira Bridge comes with several examples than can be found in
+the `examples <https://github.com/shotgunsoftware/sg-jira-bridge/tree/master/examples>`_ folder.
+
+These examples can be loaded if the `examples` directory is added to the PYTHONPATH in the
+`settings.py <https://github.com/shotgunsoftware/sg-jira-bridge/blob/master/settings.py#L95>`_ file.
+
+To properly use your custom syncer, you need to make sure you're correctly referencing it.
+This is done by using its name in the ``Jira Sync URL`` field of your Flow Production Tracking Project
+as well as in the Webhook URL on the Jira side.
+
+For example, this is how a syncer is defined in the `settings.py` file and how the urls should look like
+
+.. code-block:: python
+
+    SYNC = {
+        # Add the test syncer to the list of syncers, it will be available
+        # with the http://<your server>/jira2sg/test and http://<your server>/sg2jira/test
+        # urls.
+        "test": {
+            # Example of a custom syncer with an additional parameter to define
+            # a log level.
+            "syncer": "example_sync.ExampleSync",
+            "settings": {
+                "log_level": logging.DEBUG
+            },
+        }
+    }
+
+
+TimeLog Syncer
+**************
+
+Description
+===========
+
+The TimeLog syncer allows the synchronization of Flow Production Tracking TimeLogs with Jira Worklogs.
+It is working both way: from Flow Production Tracking to Jira and vice-versa.
+It also handles TimeLog/Worklog deletion.
+
+TimeLog Creation
+----------------
+
+When a TimeLog is created in Flow Production Tracking, an associated worklog will
+be created in Jira.
+
+.. note::
+    Only the Timelogs associated to a synced Task will be pushed to Jira. If a TimeLog is created
+    in Flow Production Tracking but is linked to a Task which don't have the ``Sync In Jira`` field checked,
+    no Jira Worklog will be created.
+
+.. warning::
+    Due to Jira API limitation, it is not possible right now to set the Worklog assignee.
+    In order to keep a track of the PTR TimeLog assignee, they are stored as JSON data in a
+    custom field on the Jira Issue.
+
+When a Worklog is created in Jira, an associated Timelog will be created in Flow Production Tracking.
+The user who created the worklog in Jira will be used as the Timelog assignee as soon as an associated
+user can be found in Flow Production Tracking.
+
+TimeLog Update
+--------------
+
+When a TimeLog is updated in Flow Production Tracking and if it has an associated Jira Worklog,
+this one will be updated too.
+
+.. warning::
+    Due to Jira API limitation, it is not possible right now to update a Worklog assignee.
+    So when a Timelog assignee is updated in Flow Production Tracking, the Jira Worklog
+    assignee won't be updated. As we are using a custom field on the Jira Issue to keep a track
+    on the TimeLog assignees, this field will be updated accordingly.
```

Comment:
I'm not sure if we need to warning twice.

Reply:
I added the warning twice in case the user decides to jump directly to a section without reading the others but I'm happy to remove it if you feel like it's too much :)